data/mods/Magiclysm/items/spell_scrolls.json

```
@@ -917,5 +917,61 @@
     "name": { "str": "Sappy Scroll" },
     "description": "This scroll is covered in sap and rose thorns.  You can still make out some of the writings, but it probably won't work the same.",
     "use_action": { "type": "learn_spell", "spells": [ "druid_tornskin" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_feralform",
+    "name": { "str": "Scroll of Feral Form" },
```

```suggestion
    "name": { "str": "Scroll of Feral Form", "str_pl": "Scrolls of Feral Form" },
```
Same for other scrolls.